x-pack/plugins/apm/public/components/app/backend_detail_operations/backend_detail_operations_list/index.tsx

```diff
@@ -0,0 +1,180 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { keyBy } from 'lodash';
+import React from 'react';
+import { useApmPluginContext } from '../../../../context/apm_plugin/use_apm_plugin_context';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import { useBreakpoints } from '../../../../hooks/use_breakpoints';
+import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
+import { useTimeRange } from '../../../../hooks/use_time_range';
+import {
+  getSpanMetricColumns,
+  SpanMetricGroup,
+} from '../../../shared/dependencies_table/get_span_metric_columns';
+import { EmptyMessage } from '../../../shared/empty_message';
+import { ITableColumn, ManagedTable } from '../../../shared/managed_table';
+import { getComparisonEnabled } from '../../../shared/time_comparison/get_comparison_enabled';
+import { TruncateWithTooltip } from '../../../shared/truncate_with_tooltip';
+import { BackendOperationDetailLink } from '../../backend_operation_detail_view/backend_operation_detail_link';
+
+interface OperationStatisticsItem extends SpanMetricGroup {
+  spanName: string;
+}
+
+function OperationLink({ spanName }: { spanName: string }) {
+  const { query } = useApmParams('/backends/operations');
+
+  return (
+    <TruncateWithTooltip
+      data-test-subj="apmOperationsListAppLink"
+      text={spanName}
+      content={<BackendOperationDetailLink {...query} spanName={spanName} />}
+    />
+  );
+}
+
+export function BackendDetailOperationsList() {
+  const {
+    query: {
+      rangeFrom,
+      rangeTo,
+      backendName,
+      environment,
+      kuery,
+      comparisonEnabled: urlComparisonEnabled,
+      offset,
+    },
+  } = useApmParams('/backends/operations');
+
+  const { core } = useApmPluginContext();
+
+  const breakpoints = useBreakpoints();
+
+  const { start, end } = useTimeRange({
+    rangeFrom,
+    rangeTo,
+  });
+
+  const comparisonEnabled = getComparisonEnabled({
+    core,
+    urlComparisonEnabled,
+  });
+
+  const primaryStatsFetch = useFetcher(
+    (callApmApi) => {
+      return callApmApi('GET /internal/apm/backends/operations', {
+        params: {
+          query: {
+            backendName,
+            start,
+            end,
+            environment,
+            kuery,
+          },
+        },
+      });
+    },
+    [backendName, start, end, environment, kuery]
+  );
+
+  const comparisonStatsFetch = useFetcher(
+    (callApmApi) => {
+      if (!comparisonEnabled) {
+        return;
+      }
+      return callApmApi('GET /internal/apm/backends/operations', {
+        params: {
+          query: {
+            backendName,
+            start,
+            end,
+            offset,
+            environment,
+            kuery,
+          },
+        },
+      });
+    },
+    [backendName, start, end, offset, environment, kuery, comparisonEnabled]
+  );
```

Comment:
Following up on @gbamparop's comment, perhaps we need  an abstraction like the one below (totally pseudo code) seeing we have comparisons in so many places now:
 
```suggestion
  const [primaryStatsFetch, comparisonStatsFetch] = useFetcherWithComparison(
    (callApmApi) => {
      return callApmApi("GET /internal/apm/backends/operations", {
        comparisonEnabled,
        params: {
          query: {
            backendName,
            start,
            end,
            offset,
            environment,
            kuery,
          },
        },
      });
    },
    [backendName, start, end, offset, environment, kuery, comparisonEnabled]
  );

```